website/docs/docs/deploy/job-commands.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Job commands"
+id: "job-commands"
+description: "How to use dbt commands to set tasks for your dbt Cloud jobs."
+---
+
+A dbt Cloud production job allows you to set up a system to run a dbt job and job commands on a schedule, rather than running dbt commands manually from the command line or [IDE](/docs/get-started/develop-in-the-cloud). Each job allows you to:
+
+- Configure job commands
+- View job run details, including timing, artifacts, and detailed run steps
+- Access logs to view or help debug issues and historical invocations of dbt
+- Set up notifications, and [more](/docs/deploy/deployments#dbt-cloud)!
+
+## Job command types
+
+Job commands are specific tasks executed by the job, and you can configure them seamlessly by either adding [dbt commands](/reference/dbt-commands) or using the checkbox option in the **Commands** section. 
+
+When using configuring job commands, you can expect different outcomes when adding the command as a run step compared to selecting a checkbox.
+
+<Lightbox src ="/img/docs/dbt-cloud/using-dbt-cloud/job-commands.gif" title="Configuring checkbox and commands list"/>
+
+
+### Built-in commands
+
+Every job invocation includes [`dbt deps`](/reference/commands/deps), meaning you don't need to add it to the **Commands** list in your job settings. `dbt deps` pulls the most recent version of the dependencies listed in your `packages.yml` from git. 
+
+**Outcome** &mdash; During a job run, the commands are “chained” together. This means if `dbt run` fails, then the subsequent commands aren't executed a and the entire job fails with an "Error" job status.
```

Comment:
hey @ernestoongaro isn't it that if `dbt deps` or `dbt run` fails, they will get the same outcome with an errored job right? if so, i propose re-adding dbt deps since that's the command we're discussing and also applies to the screenshot example

```suggestion
**Outcome** &mdash; During a job run, the commands are “chained” together. This means if `dbt deps` or `dbt run` fails, then the subsequent commands aren't executed and the entire job fails with an "Error" job status.
```

Reply:
The problem is dbt deps happens automatically, I thought it was a bad example because they didn't type the command

Reply:
oh ok, maybe i misunderstood but i thought that's what the 'built-in' commands clarification was. so built in commands = don't need to add or type it are:
- dbt deps
- dbt run
although i thought we had to type in dbt run?

Reply:
ok tried to keep the content more generic and focused on letting the user know that if there was a failure at any run step, the job would fail.